influxdb/client.py

```diff
@@ -411,9 +411,10 @@ def write_points(self,
                                       tags=tags, protocol=protocol)
 
     def ping(self):
-        """Check connectivity to influxdb
-        :returns: The version of the influxdb the client is connected to
+        """Check connectivity to InfluxDB
+        :returns: The version of the InfluxDB the client is connected to
         """
+
```

Comment:
Doh! I'll fix that, and create an issue for fixing pep257 in the remaining files